cmd/influx_inspect/verify/tsm/verify.go

```diff
@@ -46,7 +46,6 @@ func (cmd *Command) Run(args ...string) error {
 		return err
 	}
 
-	dataPath := filepath.Join(path, "data")
```

Comment:
Did you mean to include this change (would result in a backport of #23336) in this PR?

Reply:
Oh, good catch! I forced pushed a branch without the commit.